framework/web/helpers/CJavaScript.php

```diff
@@ -21,12 +21,14 @@ class CJavaScript
 	 * Quotes a javascript string.
 	 * After processing, the string can be safely enclosed within a pair of
 	 * quotation marks and serve as a javascript string.
-	 * @param string $js string to be quoted
+	 * @param string|null $js string to be quoted
```

Comment:
I would leave this as it was before - `null` is not a special case here, right now methods support everything that can be cast to string.